src/component/feedupdate/index.tsx

```diff
@@ -185,23 +175,30 @@ function FeedUpdate(props: FeedUpdateProps) {
         style: {
           zIndex: 1,
         },
-        children: updates.map((update: IUpdate) => (
-          <ContentPost
-            key={update.id}
-            update={update}
-            setIsVisible={setIsVisible}
-            isVisible={isVisible}
-            setPost={() =>
-              setPost({
+        children: updates.map((update: IUpdate) => {
+          const updateUser = allMembers.find(m => m.id === update.subjectId)
+          return (
+            <ContentPost
+              key={update.id}
+              update={{
                 ...update,
-                users: allMembers ?? [],
-              })
-            }
-            post={post}
-            allUpdates={ventureUpdates}
-            user={user}
-          />
-        )),
+                user: updateUser
+              }}
+              setIsVisible={setIsVisible}
+              isVisible={isVisible}
+              setPost={() =>
+                setPost({
+                  ...update,
+                  user: updateUser,
+                  users: allMembers ?? [],
+                })
+              }
+              post={post}
+              allUpdates={ventureUpdates}
+              user={updateUser}
```

Comment:
Here's the main fix. We need to improve the naming between the post/update user and the logged in user parameters, but this works for now.